JS_6/task2.9.js

```diff
@@ -0,0 +1,36 @@
+// 9. Написать функцию для переноса сотрудников между отделами одного предприятия. В качестве аргумента принимает два значения: id отдела, из которого будут переноситься сотрудники и id отдела, в который будут переноситься сотрудники).
+
+import { enterprises } from "./task2.1.js";
+
+// Пример:
+// moveEmployees(2, 3)
+
+const moveEmployees = ((fromDepId, toDepId) => {
+    try {
+        let fromDep, toDep;
+        enterprises.forEach(enteprise => {
+            enteprise.departments.forEach(dep => {
+                if (dep.id === fromDepId) {
+                    fromDep = dep;
+                if (dep.id === toDepId) {
+                    toDep = dep;
+                }
+                }
+            })
+        })
+        if (fromDep && toDep) {
+            toDep.employees_count += fromDep.employees_count;
```

Comment:
No check that departments are part of one enterprise

Reply:
I see